app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class Users
```

Comment:
I don't think this is how you would override #update and #destroy for users.  

Typically you would use `app/controllers/registrations_controller.rb` (no /users/) and not wrap the RegistrationsController class inside a Users class.

When I try to use the app and click the signup button I get:
```
TypeError
Users is not a class

# frozen_string_literal: true  
class Users
  class RegistrationsController < Devise::RegistrationsController     
    # before_action :configure_sign_up_params, only: [:create]     
    # before_action :configure_account_update_params, only: [:update]
```